gateway/Dockerfile.build

```diff
@@ -10,13 +10,16 @@ FROM golang:1.7.5
 
 WORKDIR /go/src/github.com/alexellis/faas/gateway
 
-COPY vendor     vendor
+COPY vendor         vendor
 
-COPY handlers   handlers
-COPY metrics    metrics
-COPY requests   requests
-COPY tests      tests
-COPY server.go  .
+COPY handlers       handlers
+COPY metrics        metrics
+COPY requests       requests
+COPY tests          tests
+COPY server.go      .
+COPY readconfig.go  .
+COPY config_test.go .
```

Comment:
@alexellis I pushed a commit that exports the fields and moves `config_test` to the *tests* package.